web/app/themes/xrnl/community_group-archive.php

```diff
@@ -16,32 +16,30 @@
 ?>
 
 <div id="community" class="container py-5">
-    <h1 class="mt-5"><?php the_title(); ?></h1>
-    <p><?php the_content(); ?></p>
+    <div class="row text-center">
+      <div class="col col-sm-11 col-md-10 col-lg-8 mx-auto">
+        <h1 class="display-3"><?php the_title(); ?></h1>
+        <p><?php the_content(); ?></p>
+      </div>
+    </div>
 
-    <div class="row">
+    <div class="row row-cols-1 row-cols-md-2 row-cols-lg-3">
       <?php while($groups->have_posts()):$groups->the_post(); ?>
-        <?php $show_link_to_group_page = !empty(get_the_content()); // Show link if group has content text ?>
+      <div class="col mb-4 mt-4">
 
-        <div class="col-8 col-sm-6 col-md-4 col-xl-4 mt-4 group-column">
-          <div id="group-card-<?php echo($groups->current_post); ?>" class="card group-card h-100 border-0">
+          <div id="group-card-<?php echo($groups->current_post); ?>" class="card group-card border-0">
 
             <?php $img_url = get_field('group_cover_image_url') ?: $default_img_url;  // Use default pic if cover image is missing ?>
-            <img src="<?php echo($img_url); ?>" class="card-img-top" alt="XRNL Community group">
-
-             <?php if($show_link_to_group_page): ?>
-               <a href="<?php the_permalink(); ?>"><h5 class="card-header font-xr text-uppercase bg-yellow border-0"><?php the_title(); ?></h5></a>
-             <?php else :?>
-              <h5 class="card-header font-xr text-uppercase bg-yellow border-0"><?php the_title(); ?></h5>
-             <?php endif; ?>
-
-              <div class="card-body bg-light text-center">
-               <?php if($show_link_to_group_page): ?>
-                 <a href="<?php the_permalink(); ?> "class="btn btn-black"><?php _e('Learn more', 'theme-xrnl'); ?></a>
+            <a href="<?php the_permalink(); ?>">
+              <img src="<?php echo($img_url); ?>" class="card-img-top" alt="XRNL Community group">
```

Comment:
I thin it's better to use the title of the group as alt text, just like in the individual page.
```suggestion
              <img src="<?php echo($img_url); ?>" class="card-img-top" alt="<?php the_title(); ?>">
```

Reply:
@SamLubbers Thanks again for your comments, great points as always :)

I've implemented your suggestions, you can you can check them out at https://development.extinctionrebellion.nl/gemeenschap/

One thing I'm not sure about is the back button. I see what you mean about top left being the usual place for it. However I think many visitors arrive at a group page not from the overview page, but from a direct link that is shared by the groups. In that context, I wonder if telling them to go "back to see all groups" is the first thing they need to see. 
The other thing is that on mobile it looks a bit awkward to me, and that it takes up a lot of premium space. I don't have any better ideas at the moment.. see what you think.

Bootstrap was updated fine, by the way :)

<img width="836" alt="image" src="https://user-images.githubusercontent.com/25393215/113787726-d5c11e80-973b-11eb-9d72-747e450b9cef.png">

Reply:
Oh and one more tiny change I made is that the main text on a group page is now left-aligned on mobile (justified didn't work well) but still justified on desktop.